Kickstarter-iOS/Views/Controllers/ManageViewPledgeRewardReceivedViewControllerTests.swift

```diff
@@ -0,0 +1,35 @@
+@testable import Kickstarter_Framework
+@testable import Library
+import UIKit
+
+final class ManageViewPledgeRewardReceivedViewControllerTests: TestCase {
+  override func setUp() {
+    super.setUp()
+
+    AppEnvironment.pushEnvironment(mainBundle: Bundle.framework)
+    UIView.setAnimationsEnabled(false)
+  }
+
+  override func tearDown() {
+    AppEnvironment.popEnvironment()
+    UIView.setAnimationsEnabled(true)
+
+    super.tearDown()
+  }
+
+  func testView() {
+    let devices = [Device.phone4_7inch, Device.phone5_8inch, Device.pad]
+    combos([Language.en], devices).forEach { language, device in
+      withEnvironment(language: language) {
+        let controller = ManageViewPledgeRewardReceivedViewController.instantiate()
+        controller.toggle.setOn(true, animated: false)
+
+        let (parent, _) = traitControllers(device: device, orientation: .portrait, child: controller)
+
+        parent.view.frame.size.height = 60
```

Comment:
More real time scenario and should make the snapshot smaller.

Reply:
I wonder if would make sense for us to wait until the manage/view pledge screen is finished in order to create snapshot test for just that viewController instead of one for each child. This way we would avoid testing the same screen twice 🤔

Reply:
Yeah, I'm not sure which side I lean towards more...I think each controller should be testable by itself but agree that as soon as we add manage pledge screen tests it will be a duplicate. 

We already do this with `SortPagerViewControllerTests` or `ProjectNavBarViewControllerTests` which test just the menu itself and then we also have a test for the container `ProjectPamphletContentViewControllerConversionTests`.

I believe testing each controller is more correct, since its parent can additionally style it or decide what it needs to do with it.